integration/init/kustomize-overlay/input/ship.yaml

```diff
@@ -0,0 +1,56 @@
+assets:
+  v1:
+    - github:
+        dest: ../base/common
+        repo: kubernetes/ingress-nginx
+        path: deploy/mandatory.yaml
+        ref: nginx-0.22.0
+        proxy: false
+        strip_path: true
+
+    - github:
+        dest: ../base
+        repo: kubernetes/ingress-nginx
+        path: deploy/provider/aws/service-l4.yaml
+        ref: nginx-0.22.0
+        proxy: false
+        strip_path: true
+
+    - github:
+        dest: ../overlays/cloud
+        repo: kubernetes/ingress-nginx
+        path: deploy/provider/aws/patch-configmap-l4.yaml
+        ref: nginx-0.22.0
+        proxy: false
+        strip_path: true
+
+    - inline:
+        dest: ./overlays/cloud/kustomization.yaml
```

Comment:
This might be worth a different issue and not addressing here. But an observation:

`github` asset types seem to have a different "root" for the `dest` attribute than `inline`?

Notice that your `github` asset types have a `dest` root of `../base`
Notice that your `inline` asset types have a `dest` root of `./overlays`

In this example, the `base` and `overlays` directories appear in the same parent, so shouldn't these share a common relative path to get there?

The difference between that `..` and `.` feels off and will likely cause errors. 

If this test is accurate, then this is an opportunity to fix or report a new issue?

Reply:
Oh wonderful, I hadn't even noticed that... And on testing, there's more to the story - empty `base` and `overlays` directories are also created in the parent directory. I'll make an issue

Reply:
Created https://github.com/replicatedhq/ship/issues/814